lib/Config.ts

```diff
@@ -32,10 +32,15 @@ class Config {
   public noencrypt = true; // TODO: enable encryption by default
 
   /**
-   * Whether to disable sanity checks that verify that the orders can possibly be swapped
-   * before adding them to the order book, can be enabled for testing & debugging purposes.
+   * Whether to enable sanity swaps that verify that the orders can possibly be swapped
+   * before adding trading pairs as active.
    */
-  public nosanitychecks = false;
+  public sanityswaps: boolean;
```

Comment:
Some of the other boolean config options have their default value set here where they are declared (like `nomatching` above). It's less code that way, although I suppose the benefit of setting all default values in the constructor is that they're all in one place, since some values can't be initialized outside the constructor. I think I prefer having default values where they're initialized (like `public sanityswaps = true;`) where it's possible to do so, but either way I think it's best to be consistent, just wondering what you think is best going foward.

Reply:
I agree - let's be consistent. Think your way of doing it is better because it's less verbose.